MapboxVision/src/test/java/com/mapbox/vision/telemetry/TotalBytesCounterTest.kt

```diff
@@ -0,0 +1,275 @@
+package com.mapbox.vision.telemetry
+
+import com.mapbox.vision.utils.system.Time
+import io.mockk.every
+import io.mockk.mockk
+import org.junit.jupiter.api.Assertions.*
+import org.spekframework.spek2.Spek
+import org.spekframework.spek2.style.gherkin.Feature
+import java.util.concurrent.TimeUnit
+
+const val TEST_SESSION_MAX_BYTES = 10_000L
```

Comment:
do we need these properties in `public`?

Reply:
right now I haven't got solution. let's discuss it

Reply:
Just move it inside the object?

Reply:
done